doc/cephfs/hadoop.rst

```diff
@@ -30,7 +30,7 @@ and configure Hadoop to use the Ceph file system.
 CephFS Java Packages
 --------------------
 
-* CephFS Hadoop plugin (`hadoop-cephfs.jar <http://ceph.com/download/hadoop-cephfs.jar>`_)
+* CephFS Hadoop plugin (`hadoop-cephfs.jar <https://download.ceph.com/tarballs/hadoop-cephfs.jar>`_)
```

Comment:
there are a few more of these 'ceph.com/download/' links, would you mind updating those too?

    grep -Hrn ceph\.com\/download *
    cephfs/hadoop.rst:33:* CephFS Hadoop plugin (`hadoop-cephfs.jar <http://ceph.com/download/hadoop-cephfs.jar>`_)
    dev/release-process.rst:130:	     ceph_site@ceph.com:ceph.com/downloads/.
    dev/release-process.rst:137:            ceph_site@ceph.com:ceph.com/downloads/ceph-0.xx.tar.bz2
    dev/release-process.rst:139:            ceph_site@ceph.com:ceph.com/downloads/ceph-0.xx.tar.gz
    install/build-ceph.rst:85:	wget -P ~/rpmbuild/SOURCES/ http://ceph.com/download/ceph-<version>.tar.bz2
    install/build-ceph.rst:89:	wget -P ~/rpmbuild/SOURCES/ http://eu.ceph.com/download/ceph-<version>.tar.bz2
    start/get-involved.rst:68:|                      | the very latest code for Ceph, you can get it   | - http://ceph.com/download                    |

Reply:
@alfredodeza i don't think the `ceph_site@ceph.com:ceph.com/downloads`s are links. they are more likely ssh paths. and i don't have an account on ceph.com to verify this. so i'd like to keep them intact.

i change the links in the command line and the "watch" URLs in ceph.spec and debian/watch.